bids-validator/tests/json.spec.js

```diff
@@ -467,6 +467,80 @@ describe('JSON', function() {
     relativePath: '/dataset_description.json',
   }
 
+  it('dataset_description.json should validate DatasetLinks', function() {
+    var jsonObj = {
+      Name: 'Example Name',
+      BIDSVersion: '1.4.0',
+      DatasetLinks: {
+        mylink: 'https://www.google.com',
```

Comment:
Let's check that the example URIs we list in the spec PR don't cause errors:

```suggestion
        mylink: 'https://www.google.com',
        deriv1: 'derivatives/derivative1',
        phantoms: 'file:///data/phantoms',
        ds000001: 'doi:10.18112/openneuro.ds000001.v1.0.0',
```